husky_components/datagrid/decorators/table-view.js

```diff
@@ -286,10 +292,29 @@ define(function() {
         },
 
         /**
+         * Indents the datagrid's selected-counter
+         */
+        indentSelectedCounter: function() {
+            if (this.datagrid.options.selectedCounter === true) {
+                this.sandbox.dom.addClass(this.datagrid.$find('.selected-elements'), constants.counterIndentClass);
+            }
+        },
+
+        /**
+         * Removes the indent of the selected-counter
+         */
+        removeIndentSelectedCounter: function() {
+            if (this.datagrid.options.selectedCounter === true) {
+                this.sandbox.dom.removeClass(this.datagrid.$find('.selected-elements'), constants.counterIndentClass);
+            }
+        },
+
+        /**
          * Destroys the view
          */
         destroy: function() {
             this.sandbox.stop(this.sandbox.dom.find('*', this.$el));
+            this.removeIndentSelectedCounter();
```

Comment:
the view gets destroyed. but i added the select-counter to the main-class as it's needed in other views as well. in the case where the table-view takes up the whole width a had to give the counter a little indent on the right as it looked horrible without. Therefore i have to remove the indent after the table-view gets destroyed. These things are the purpose of the destory-methods of views